### DISCUSSION TITLE
Opacity for SVG Editor

### DISCUSSION THREAD

**Discussion by DabDatBass** [ScratchAddons/ScratchAddons#1384] [Ideas]
Hello!

Fun fact: Opacity can be loaded into the Scratch SVG Editor. Another fun fact: Opacity isn't implemented yet.

So, maybe a slider to change the opacity? (And if hex color picker for the editor is on it can add onto the hex code, such as #008080 would become #0080804b)

**Comment by TheColaber** [comment]
nvm its hard
